packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -26,6 +26,7 @@ describe('network stubbing', { retries: 15 }, function () {
 
   beforeEach(function () {
     cy.spy(Cypress.utils, 'warning')
+    cy.visit('/fixtures/empty.html')
```

Comment:
with Electron 28, we need to visit a url to establish origin , even though the `document.location` is already on the correct origin. A bit strange, but this does make sense for security purposes. I do not think this is a breaking change but more so an oddity with the test.

Reply:
What is the error when this happens?

Reply:
Will this be technically breaking? Folks could have tests that could have worked previously without a visit that no longer work?

Reply:
The error was fairly vague via `ERR_INVALID_ARGUMENT` when an xhr request would fire. The only way I could resolve it was to visit and establish the origin before making requests to that origin server.  By guess is it was a problem with the domain not being established while attempting to make same origin connections. This could technically be breaking, I wonder for those doing API testing, but I would also think most people doing API testing are doing so from cypress commands through the node API and not browser requests before visiting a page